### PR TITLE
#5629 chiller electric eir air/evap cooled condenser outlet node conditions not correct and evap condenser water use = 0

### PR DESCRIPTION
**Pull request by rraustad**
## Pull request overview

Air-cooled and evaporatively-cooled EIR chillers reports very high condenser air outlet node temperatures. Evap-cooled chiller condenser also shows no water use. Fixes issue #5629.
### Work Checklist

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
- [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
- [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
  - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
### Review Checklist

This will not be exhaustively relevant to every PR.
- [x] Code style (parentheses padding, variable names)
- [x] Functional code review (it has to work!)
- [x] If defect, results of running current develop vs this branch should exhibit the fix
- [ ] CI status: all green or justified
- [ ] Performance: CI Linux results include performance check -- verify this
- [x] Unit Test(s)
- C++ checks:
  - [ ] Argument types
  - [ ] If any virtual classes, ensure virtual destructor included, other things
- IDD changes:
  - [ ] Verify naming conventions and styles, memos and notes and defaults
  - [ ] Open windows IDF Editor with modified IDD to check for errors
  - [ ] If transition, add rules to spreadsheet
  - [ ] If transition, add transition source
  - [ ] If transition, update idfs
- [ ] If new idf included, locally check the err file and other outputs
- [x] Documentation changes in place
- [ ] ExpandObjects changes??
- [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

File: doc/input-output-reference/src/overview/group-plant-equipment.tex [hunk 1]
```diff
@@ -1772,6 +1776,12 @@ \subsubsection{Outputs}\label{outputs-4-010}
 
 These outputs are the heat transfer rate and total heat transfer due to false loading of the chiller. When the chiller part-load ratio is below the minimum unloading ratio, the chiller false loads (e.g.~hot-gas bypass) to further reduce capacity. The false load heat transfer output variable is not metered.
 
+\paragraph{Chiller Evaporative Condenser Water Volume {[}m3{]}}\label{chiller-evaporative-condenser-water-volume-m3}
+
+\paragraph{Chiller Evaporative Condenser Mains Supply Water Volume {[}m3{]}}\label{chiller-evaporative-condenser-mains-supply-water-volume-m3}
+
+These outputs are the water use for evaporatively-cooled condensers. When the chiller operates, the water consumed by the evaporatively-cooled condenser is proportional to the chiller condenser heat transfer. The evaporative condenser is assumed to be 100% effective where the condenser inlet air dry-bulb temperature is equal to the outdoor wet-bulb temperature. 
+
```
Comment (rraustad) [inline doc/input-output-reference/src/overview/group-plant-equipment.tex:1784]: So the dimmed characters on my screen were real. I guess the % needs some type of delimiter. I see \% on Google.

**Comment by rraustad** [comment]
Corrected the field name for the EIR chiller so there will be diff's in a lot of eio files.
Changed "Water" to "Fluid" to match IDD.

```
- Component Sizing Information, Chiller:Electric:EIR, MAIN CHILLER, Design Size Reference Condenser Water Flow Rate [m3/s], 1.81478E-003
+ Component Sizing Information, Chiller:Electric:EIR, MAIN CHILLER, Design Size Reference Condenser Fluid Flow Rate [m3/s], 1.81478E-003
```

**Comment by rraustad** [comment]
Reviewed 36 eio files. 

1 file has extra eio reports for the HX (HVACStandAloneERV_Economizer)
31 have diff's in name only where "Water" as changed to "Fluid". No diff's in autosized result. 

4 have diff's in autosize result for air-cooled chillers where result now reflects air flow rate instead of water flow rate.
These 4 files also had a blank "chiller type" as:

DOASDualDuctSchool:

```
- Component Sizing Information, Chiller:Electric:EIR, COOLSYS1 CHILLER, User-Specified Reference Condenser Water Flow Rate [m3/s], 1.10000E-003
+ Component Sizing Information, , COOLSYS1 CHILLER, Design Size Reference Condenser Fluid Flow Rate  [m3/s], 52.37477
```

The 4 files with blank chiller type are:
DOASDualDuctSchool
HVACTemplate-5ZoneUnitarySystem
RefBldgLargeHotelNew2004_Chicago
RefBldgSecondarySchoolNew2004_Chicago

Expecting 1 less eio diff for HVACStandAloneERV_Economizer where this branch was behind develop.
Expecting the 4 files with blank chiller type to now show chiller type in eio.

**Comment by rraustad** [comment]
The 1 AUD diff should also be gone with this new check in since it reflected extra lines for HVACStandAloneERV_Economizer

**Comment by rraustad** [comment]
StandaloneERV file is now not shown as a file with eio diff's.
All 4 files with missing chiller type are now correct.
This is ready for review.

**Comment by Myoldmopar** [comment]
Yeah, this is great.  I verified the fix by running DOASDualDuctSchool (the only example file with an air-cooled EIR chiller methinks) with additional outputs.  The outlet temperature looks yeah a smidge better now 😉 .  Code changes and unit tests are great.  No CI issues.  Thanks @rraustad.

**Pull request merged**